xray/src/generation.rs

```diff
@@ -300,17 +365,23 @@ impl ColoringStrategy for PointColorColoringStrategy {
                 {
                     Entry::Occupied(mut e) => {
                         let per_column_data = e.get_mut();
-                        per_column_data.color_sum.red += clr.red;
-                        per_column_data.color_sum.green += clr.green;
-                        per_column_data.color_sum.blue += clr.blue;
-                        per_column_data.color_sum.alpha += clr.alpha;
-                        per_column_data.count += 1;
+                        let data = per_column_data.entry(bins[i]).or_insert(PerColumnData {
```

Comment:
See my comments about `IntensityColorignStrategy`

Reply:
I changed it accordingly.